engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties

```diff
@@ -1,40 +1,53 @@
+org.hibernate.validator.constraints.NotEmpty.message                = darf nicht leer sein.
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = Die Evaluierung des Scriptausdrucks "{script}" ergab nicht true.
+org.hibernate.validator.constraints.Range.message                   = muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.SafeHtml.message                = hat m\u00f6glicherweise keinen gesch\u00fctzten HTML-Inhalt.
+org.hibernate.validator.constraints.ScriptAssert.message            = Die Evaluierung des Scriptausdrucks "{script}" ergab nicht true.
+org.hibernate.validator.constraints.UniqueElements.message          = dar nur eindeutige Elemente enthalten
+org.hibernate.validator.constraints.URL.message                     = muss eine g\u00fcltige URL sein.
 
-org.hibernate.validator.constraints.time.DurationMax.message        = muss k\u00FCrzer${inclusive == true ? ' oder gleich' : ' als'}${days == 0 ? '' : days == 1 ? ' 1 Tag' : ' ' += days += ' Tage'}${hours == 0 ? '' : hours == 1 ? ' 1 Stunde' : ' ' += hours += ' Stunden'}${minutes == 0 ? '' : minutes == 1 ? ' 1 Minute' : ' ' += minutes += ' Minuten'}${seconds == 0 ? '' : seconds == 1 ? ' 1 Sekunde' : ' ' += seconds += ' Sekunden'}${millis == 0 ? '' : millis == 1 ? ' 1 Millisekunde' : ' ' += millis += ' Millisekunden'}${nanos == 0 ? '' : nanos == 1 ? ' 1 Nanosekunde' : ' ' += nanos += ' Nanosekunden'} sein
-org.hibernate.validator.constraints.time.DurationMin.message        = muss l\u00E4nger${inclusive == true ? ' oder gleich' : ' als'}${days == 0 ? '' : days == 1 ? ' 1 Tag' : ' ' += days += ' Tage'}${hours == 0 ? '' : hours == 1 ? ' 1 Stunde' : ' ' += hours += ' Stunden'}${minutes == 0 ? '' : minutes == 1 ? ' 1 Minute' : ' ' += minutes += ' Minuten'}${seconds == 0 ? '' : seconds == 1 ? ' 1 Sekunde' : ' ' += seconds += ' Sekunden'}${millis == 0 ? '' : millis == 1 ? ' 1 Millisekunde' : ' ' += millis += ' Millisekunden'}${nanos == 0 ? '' : nanos == 1 ? ' 1 Nanosekunde' : ' ' += nanos += ' Nanosekunden'} sein
+org.hibernate.validator.constraints.br.CNPJ.message                 = Ung\u00fcltige brasilianische Registriernummer f\u00fcr K\u00f6rperschaftssteuerzahlungen (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = Ung\u00fcltige brasilianische Registriernummer f\u00fcr Steuerzahlungen nat\u00fcrlicher Personen (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = Ung\u00fcltige brasilianische ID-Karte f\u00fcr Entscheidungsberechtigte
+
+org.hibernate.validator.constraints.pl.REGON.message                = Ung\u00fcltige polnische Steuerzahleridentifikationsnummer (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = Ung\u00fcltige Mehrwertsteueridentifikationsnummer (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = Ung\u00fcltige polnische nationale Identifikationsnummer (PESEL)
```

Comment:
I believe `Ung\u00fcltige` is not a noun so it shouldn't be capitalized if we want to be consistent.